manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer-windows.yaml

```diff
@@ -61,6 +61,13 @@ spec:
             periodSeconds: 30
           resources:
 {{ toYaml .Values.windows.resources | indent 12 }}
+          {{- if .Values.enableArcExtension }}
+          {{- if .Values.Azure.proxySettings.isProxyEnabled }}
```

Comment:
We need to add this change too? https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/e11dd11955657b4a731a0ecfeb3ad6bc2127f723/manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer.yaml#L31-L37

Reply:
This is to detect cert file changes. We are not supporting it as a part of this change. So I skipped it.